developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,428 @@ title: "Mint an NFT with Aptos CLI"
 slug: "mint-nft-cli"
 ---
 
-# Mint an NFT
+# Mint an NFT with Aptos CLI
+
+This tutorial lets you use the Aptos CLI to mint non-fungible tokens (NFTs) in Aptos testnet so you can see how the process works and employ related functions in your code.
 
 ## Prerequisites
 
 This tutorial assumes you have:
 
-* a GitHub account
-* the GitHub CLI
-* the Aptos CLI (installed below)
+* a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
+* the [GitHub CLI](https://cli.github.com/)
+* the [Aptos CLI](../../cli-tools/aptos-cli-tool/install-aptos-cli.md) (or you can run from [aptos-core](https://github.com/aptos-labs/aptos-core) source via `cargo run`)
+* the `aptos-core` repository checked out: `git clone https://github.com/aptos-labs/aptos-core.git`
+
+## Understand minting
+
+TODO: Consider moving all of this conceptual information to its own page, perhaps a new NFT index. Merge with conceptual information from Workshop #2.
+
+* Review the [mint_nft](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/mint_nft) source code and code comments within each subdirectory.
+* Explore the `mint_event_ticket` function defined within each subdirectory.
+
+### NFT types
+
+The two most common types of NFT are Event ticket / certificates and PFP NFTs.
+
+#### Event tickets and certificates
```

Comment:
this is an awesome explanation!!!!!

Reply:
Thanks! I harvested from your code comments. :-)